CMakeLists.txt

```
@@ -222,6 +222,9 @@ else()
 
 endif()
 
+# Generate .NET Version file.
+configure_file(VersionInfo.cs.in ${CMAKE_CURRENT_SOURCE_DIR}/src/csharp/VersionInfo.cs)
```

In order to keep the build tree clean can you put this in `${PROJECT_BINARY_DIR}/src/csharp/VersionInfo.cs`.

This will ensure that 1) someone doesn't accidentally check in VersionInfo.cs and 2) prevent git from thinking there is a modified file in its repo when it starts building

```suggestion
configure_file(VersionInfo.cs.in ${PROJECT_BINARY_DIR}/src/csharp/VersionInfo.cs)
```

---

I'd rather keep it with the source to make it easier to find the files and prevent confusion over this VersionInfo.cs changing location depending on building x86 vs x64.